### PR TITLE
Fixing aria/flask dependency break

### PR DESCRIPTION
**Pull request by tonytan4ever**

### CODE REVIEW

**Comment by prometheanfire** [comment]
might work, not sure as it's OSA's gate-check-commit that's installing it, not maas

**Comment by odyssey4me** [comment]
ceph-deploy failure:

```
23:56:41 Running virtualenv with interpreter /var/lib/jenkins/workspace/PR_rpc-maas-master-xenial-ceph-deploy/rpc-maas/.tox/functional/bin/python
23:56:41 Using real prefix '/usr'
23:56:41 New python executable in /opt/test-maas/bin/python
23:56:44 Installing setuptools, wheel...done.
23:56:46 Collecting pip
23:56:47   Using cached https://files.pythonhosted.org/packages/0f/74/ecd13431bcc456ed390b44c8a6e917c1820365cbebcb6a8974d1cd045ab4/pip-10.0.1-py2.py3-none-any.whl
23:56:48 Installing collected packages: pip
23:56:48   Found existing installation: pip 9.0.3
23:56:48     Not uninstalling pip at /var/lib/jenkins/workspace/PR_rpc-maas-master-xenial-ceph-deploy/rpc-maas/.tox/functional/lib/python2.7/site-packages, outside environment /opt/test-maas
23:56:48     Can't uninstall 'pip'. No files were found to uninstall.
23:56:48 Successfully installed pip-10.0.1
23:56:48 You are using pip version 9.0.3, however version 10.0.1 is available.
23:56:48 You should consider upgrading via the 'pip install --upgrade pip' command.
23:56:49 Traceback (most recent call last):
23:56:49   File "/opt/test-maas/bin/pip", line 7, in <module>
23:56:49     from pip._internal import main
23:56:49 ImportError: No module named _internal
```

**Comment by odyssey4me** [comment]
liberty-deploy failure:

```
00:12:18 TASK: [galera_server | Install galera packages] ******************************* 
00:12:33 failed: [aio1_galera_container-5908bd29] => (item=mariadb-client,mariadb-galera-server-10.0,galera-3,qpress,rsync,socat) => {"attempts": 5, "failed": true, "item": "mariadb-client,mariadb-galera-server-10.0,galera-3,qpress,rsync,socat"}
00:12:33 msg: Task failed as maximum retries was encountered
00:12:33 
00:12:33 msg:
00:12:33 Task failed as maximum retries was encountered
00:12:33 
00:12:33 FATAL: all hosts have already failed -- aborting
```

**Comment by tonytan4ever** [comment]
recheck_trusty_liberty_deploy

**Comment by mattt416** [comment]
recheck_trusty_kilo_deploy

File: tests/aio-create.sh [hunk 1]
```diff
@@ -213,6 +213,8 @@ pushd /opt/openstack-ansible
 
   elif [ "${RE_JOB_SCENARIO}" == "pike" ]; then
     git checkout "stable/pike"  # Branch checkout of Pike (Current Stable)
+    # Pin flask so it stops breaking xenial and other versions
+    echo "Flask==0.12.2" >> /opt/openstack-ansible/global-requirement-pins.txt
```
Comment (prometheanfire) [inline tests/aio-create.sh]: didn't we want to remove this?

Comment (tonytan4ever) [reply tests/aio-create.sh]: I tried to remove this and upper-constraint in tox ini,  it breaks all other gates.
I'd like to start a new overhaul to remove all tox env vars.

**Comment by tonytan4ever** [comment]
Pike fixed, merging.

**Pull request merged**